index.html

```diff
@@ -191,7 +383,7 @@ <h3>
                 <li class="s4"><img src="images/twitter.png" alt="twitter"></li>
                 <li class="s5"><img src="images/em.png" alt="Media"></li>
             </ul>
-    
+            
         <div class="div-low"></div>
```

Comment:
### Your Black Bar
![black-bar-position](https://user-images.githubusercontent.com/93039359/177856940-b8b12b99-9c4d-4972-9b0d-36d6d26347af.png)

- [ ] Green box shows the correct position of your black bar. Please make changes in your margin as shown in the quick fix below :+1:
### **Quick Fix**
![black-bar-styling](https://user-images.githubusercontent.com/93039359/177856806-1e0d90a6-8b19-4eb4-b84d-6c4e225df921.png)